basic/image_basic.py

```diff
@@ -109,26 +109,100 @@ def channel_lim(value):
         return value / 255
 
     @staticmethod
-    def to_array(image, array_type='uint8'):
+    def show_image_gallery(title, images, rows, columns, shape=()):
         """
-        Convert a PIL Image object
-        to numpy array
+        Plot or save two or more (processed) images
 
         Parameters
         ----------
-        image: PIL Image Object
-            An instance of a image object from PIL module
-        array_type: str
-            Numpy type for the array
+        title: str
+            Title of the plot (which will appear top centered)
+        images: numpy.ndarray
+            A numpy array containing image data
+        rows: int
+            Number of rows to plot
+        columns: int
+            Number of columns to plot
+        shape: tuple
+            A tuple containing the dimensions of image
+            (used for reshaping the array if it was flatten)
+        """
+        plot.suptitle(title, size=30)
+
+        for index, image in enumerate(images):
```

Comment:
Verify if `images` array has more than one array in it (since this method must plot two or more images in a gallery).